### PR TITLE
updating mutators documentation

### PR DESCRIPTION
**Pull request by lkwg82**
no numbers means no wrong numbers ;)

### CODE REVIEW

File: quickstart/mutators.markdown [hunk 1]
```diff
@@ -744,85 +744,6 @@ mutations of void methods and
...
-
-will be mutated to
-
-<pre class="prettyprint lang-java">
-short s = Byte.MIN_VALUE;
-int i = Short.MIN_VALUE;
-int j = Byte.MIN_VALUE
-</pre>
-
-<a name="EXPERIMENTAL_MEMBER_VARIABLE" id="EXPERIMENTAL_MEMBER_VARIABLE"></a>
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown:825]: You've killed the anchor or the next one ;-)

File: quickstart/mutators.markdown [hunk 2]
```diff
@@ -25,21 +22,24 @@ more detailed examples.
 
 Here is the list of available mutators:
 
+(*activated by default*)
 - [Conditionals Boundary Mutator](#CONDITIONALS_BOUNDARY)
-- [Negate Conditionals Mutator](#NEGATE_CONDITIONALS)
-- [Remove Conditionals Mutator](#REMOVE_CONDITIONALS)
-- [Math Mutator](#MATH)
+- [Constructor Calls Mutator](#CONSTRUCTOR_CALLS)
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown]: This one is not active by default

File: quickstart/mutators.markdown [hunk 3]
```diff
@@ -25,21 +22,24 @@ more detailed examples.
...
 - [Increments Mutator](#INCREMENTS)
-- [Invert Negatives Mutator](#INVERT_NEGS)
 - [Inline Constant Mutator](#INLINE_CONSTS)
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown]: Inline Constant Mutator is not active by default

File: quickstart/mutators.markdown [hunk 4]
```diff
@@ -25,21 +22,24 @@ more detailed examples.
...
+- [Invert Negatives Mutator](#INVERT_NEGS)
+- [Negate Conditionals Mutator](#NEGATE_CONDITIONALS)
+- [Non Void Method Calls Mutator](#NON_VOID_METHOD_CALLS)
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown]: Non-Void-Method-Call is not active by default

File: quickstart/mutators.markdown [hunk 5]
```diff
@@ -25,21 +22,24 @@ more detailed examples.
...
+- [Math Mutator](#MATH)
+- [Remove Conditionals Mutator](#REMOVE_CONDITIONALS)
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown]: remove conditionals is not active by default but partially included in stronger group

File: quickstart/mutators.markdown [hunk 6]
```diff
@@ -25,21 +22,24 @@ more detailed examples.
...
 - [Return Values Mutator](#RETURN_VALS)
 - [Void Method Calls Mutator](#VOID_METHOD_CALLS)
-- [Non Void Method Calls Mutator](#NON_VOID_METHOD_CALLS)
-- [Constructor Calls Mutator](#CONSTRUCTOR_CALLS)
-- [Experimental Inline Constant Mutator](#EXPERIMENTAL_INLINE_CONSTS)
+
+(*deactivated by default*)
 - [Experimental Member Variable Mutator](#EXPERIMENTAL_MEMBER_VARIABLE)
 - [Experimental Switch Mutator](#EXPERIMENTAL_SWITCH)
```
Comment (StefanPenndorf) [inline quickstart/mutators.markdown:40]: experimental switch mutator is member of stronger group

Note: There is also a remove switch mutator which isn't documented at all at the moment.

**Comment by lkwg82** [comment]
ping

**Comment by hcoles** [comment]
@lkwg82 Sorry, merging now.

**Pull request merged**